tableau-databricks/connection-builder.js

```diff
@@ -40,6 +40,9 @@ limitations under the License.
 	// Minimum interval between consecutive polls for query execution status (1ms)
 	params["AsyncExecPollInterval"] = "1";
 
+	// Tell the ODBC driver that it is Tableau connecting.
+	params["UserAgentEntry"] = "Tableau";
```

Comment:
@pvanderknyff could I possibly get the Tableau version programatically from here?
If possible, I would like to pass the Tableau version here, like "Tableau/2019.4", but not hardcode it for every release.
Is that possible?

Reply:
Unfortunately, that is not possible at the moment. Can you open an enhancement request on our [github issues page](https://github.com/tableau/connector-plugin-sdk/issues) for a connection helper so we can look at maybe putting such a feature on our backlog? Thanks.